src/pkg/packager/publish.go

```diff
@@ -48,26 +50,36 @@ func (p *Packager) Publish(ctx context.Context) (err error) {
 	}
 
 	if p.cfg.CreateOpts.IsSkeleton {
```

Comment:
Does this have to be changed?

Reply:
No we can leave changing this until after publish has been refactored if you prefer.

Reply:
I think that would be best